lib/recording/RecordingManager.js

```diff
@@ -43,119 +44,136 @@ class RecordingManager {
 
         logger.info("Initializing recording manager");
 
-        setInterval(()=>{
+        setInterval(()=> {
             this.handleRecordingEntries()
-        }, recordingTimeInterval, this)
+        }, recordingTimeInterval, this);
     }
 
     startRecording(entryObj, flavorsObjArray){
         return Q.resolve().then(()=> {
```

Comment:
General comment: if startRecording returned promise is rejected live entry will fail to start. I don't think we want that but is for product to decide. Please consult with Guy & Yaron and test the reject flow as well.